ruby-sinatra/simapi/sim_api.rb

```diff
@@ -4,10 +4,16 @@
 require "sinatra"
 require "json"
 require "newrelic_rpm"
+require 'dotenv'
+
```

Comment:
Not your fault, but something to consider for the future:
* We might want to rename this folder, as the current naming is not very clear. Something like `api` or `controllers` could be better?
* We could also consider separating the routes into different controllers, for example, a controller for users, another for messages, etc. - think it's convention for sinatra and ruby but don't know 100%